ivy/array/elementwise.py

```diff
@@ -10,13 +10,32 @@
 
 # noinspection PyUnresolvedReferences
 class ArrayWithElementwise(abc.ABC):
-    def abs(self: ivy.Array, out: Optional[ivy.Array] = None) -> ivy.Array:
+    def abs(
```

Comment:
Seems like you've added an `x1` argument to the functions, but we don't need that argument, the `abs` function simply accepts two arguments, `x` which is the input argument and `out` which is the out argument, please remove the `x1` argument.